### PR TITLE
Merged Links With Text in README

### PR DESCRIPTION
**Pull request by nishanth1232**
The links in the README have been merged with the text using Markdown.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +108,7 @@ You can also run `./ora.sh help` to see more commands for administration of the
 
 #### Step 4: Accessing your programming environment
 
-Start your browser at: http://localhost:1999
+Start your browser [here](http://localhost:1999).
```
Comment (boonto) [inline README.md]: Can you change this to explicitly state `http://localhost:1999`, that information should be included and not hidden by the link.

File: README.md [hunk 2]
```diff
@@ -143,7 +141,7 @@ Furthermore, the project OpenRobertaServer contains in directory staticResources
 * HTML and CSS
 * Javascript libraries based on jquery and bootstrap for the frontend
   * assertions (DBC), ajax-based server calls (COMM), logging (LOG) and
-  * javascript resources for blockly (see: http://code.google.com/p/blockly/)
+  * javascript resources for [blockly](http://code.google.com/p/blockly/)
```
Comment (boonto) [inline README.md]: Please use the updated link from the other merged PR here.

File: README.md (hunk 1 above)
Comment (nishanth1232) [reply README.md]: Made the changes.

File: README.md (hunk 2 above)
Comment (nishanth1232) [reply README.md]: Made the changes.

**Comment by boonto** [comment]
Thanks! Merged in f4ba640c7c3ae8d9608d323d7ce34ae683aaa421.

**Pull request closed without merging**